Exercise_C/index.css

```diff
@@ -2,3 +2,39 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+.container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    width: 100px;
+    height: 220px;
+
+}
+
+.box1 {
+    background-color: blue;
+    width: 100px;
+    height: 100px;
+    margin: 10px;
+    display: flex;
+    justify-self: flex-start;
+}
+
+.box2 {
+    background-color: red;
+    display: inline-flex;
+    width: 100px;
+    height: 100px;
+    align-self: flex-end;
+    margin: 10px;
+
+}
+
+.box3 {
+    background-color: green;
+    width: 100px;
+    height: 100px;
+    margin: 10px;
+
```

Comment:
Again extract you properties so you do not need to repeat code.
Decent use of flex box to achieve this work

Another implementation would required you to alter your HTML two have two containers for the blue and green square, and the other container for the red square.